packages/react-graphql-universal-provider/src/test/GraphQLUniversalProvider.test.tsx

```diff
@@ -29,55 +29,27 @@ jest.mock('@shopify/react-graphql', () => {
 
 describe('<GraphQLUniversalProvider />', () => {
   it('renders an ApolloProvider with a client created by the factory', () => {
-    const client = new ApolloClient({
+    const clientOptions = {
       cache: new InMemoryCache(),
       link: new ApolloLink(),
-    });
+    };
     const graphQL = mount(
-      <GraphQLUniversalProvider createClient={() => client} />,
+      <GraphQLUniversalProvider createClientOptions={() => clientOptions} />,
     );
 
     expect(graphQL).toContainReactComponent(ApolloProvider, {
       client: expect.any(ApolloClient),
     });
   });
 
-  it('serializes the apollo apollo cache and re-uses it to hydrate the cache', async () => {
```

Comment:
This test called `client.extract()` manually which is no longer accessible. Is there any other way we can test this functionality still?

Reply:
cc/ @michenly

Reply:
couldn't we do it the same way that `react-server` does? ie using `extract` from `react-effect/server`?

Reply:
maybe we can try to get `client` from `ApolloProvider`'s prop for extraction?

Reply:
Michelle's suggestion worked. Pulled client from the `ApolloProvider`'s props.